sympy/printing/octave.py

```diff
@@ -441,10 +456,30 @@ def _print_airybiprime(self, expr):
         return "airy(3, %s)" % self._print(expr.args[0])
 
 
-    def _print_LambertW(self, expr):
-        # argument order is reversed
-        args = ", ".join([self._print(x) for x in reversed(expr.args)])
-        return "lambertw(" + args + ")"
+    def _print_expint(self, expr):
+        mu, x = expr.args
+        if mu != 1:
+            # they only implement nu = 1
```

Comment:
should `nu` here be `mu`?

Reply:
Thanks!  Maybe that comment is self-evident from the code; I will remove it.